src/Sparrow.Server/Meters/IoMetrics.cs

```diff
@@ -10,12 +10,12 @@ public enum MeterType
         {
             Compression,
             JournalWrite,
+            JournalWait,
```

Comment:
We need to check if this requires any adjustments in Studio IO Metrics graph

Reply:
Yes, who would be the one to hand over this ticket when this is done, or do I create a new ticket?

Reply:
Assign to @arekpalinski he will distrubte